packages/sdc/components/03-organisms/promo/promo.css

```
@@ -29,18 +29,19 @@
   }
 }
 .ct-promo__links {
-  margin-top: 1.5rem;
+  margin-top: 1rem;
 }
 @media (min-width: 768px) {
   .ct-promo__links {
     margin-top: 0;
   }
 }
```

Stylelint (17.14.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)